src/libcore/lib.rs

```diff
@@ -121,7 +121,7 @@
 #![feature(const_slice_len)]
 #![feature(const_str_as_bytes)]
 #![feature(const_str_len)]
-#![feature(const_let)]
+#![cfg_attr(stage0, feature(const_let))]
```

Comment:
libcore now doesn't need this feature gate anymore -> some more const fns could be made stable after this PR